### ISSUE TITLE
Module not found: Error: Can't resolve 'angular-instantsearch/clear-refinements/clear-refinements'

### ISSUE THREAD

**Issue by corentin-gautier** [algolia/angular-instantsearch#250]
Hi,

When trying to import NgAisClearRefinements or NgAisRefinementList or basically every sub components of angular-instantsearch I get a can't resolve error. The only way I have found to import them is to import them like this: 

``` typescript
import { ɵo as NgAisRefinementList } from 'angular-instantsearch'
```

Am I missing something ?

**Comment by Haroenv** [comment]
You seem to be right, I didn't know our build tool (ng-packagr) does that transformation. While this is something that we should probably fix, in the mean time you can find the name mapping here: https://cdn.jsdelivr.net/npm/angular-instantsearch@2.0.4/angular-instantsearch.d.ts

**Comment by egorkel-da14** [comment]
Not fixed in 2.3.0
Have to use this
`import { ɵr as NgAisSearchBox } from 'angular-instantsearch';`

**Comment by rubenheymans** [comment]
this is still not fixed?
